docs/software/ml/tutorials/llm-nanotron-training.md

```diff
@@ -1,17 +1,20 @@
-[](){#ref-mlp-llm-nanotron-tutorial}
+[](){#software-ml-llm-nanotron-tutorial}
 
 # LLM Nanotron Pre-training Tutorial
 
 In this tutorial, we will build a container image to run multi-node training jobs with [nanotron](https://github.com/huggingface/nanotron).
 We will train a 109M parameter model with ~100M wikitext tokens as a proof of concept.
 
+!!! info
+    Note that while the concepts taught here for multi-node training with PyTorch are generally portable across frameworks, the current (August 2025) recommendation for users with a need for large-scale model-parallel training is to use `Megatron-LM` instead of `nanotron` due to significant performance advantages at scale. 
+
 ### Prerequisites 
 
-It is recommended to follow the previous two tutorials on [LLM Inference][ref-mlp-llm-inference-tutorial] and [LLM Fine-tuning][ref-mlp-llm-fine-tuning-tutorial] first, as this will build upon them.
+It is recommended to follow the previous two tutorials on [LLM Inference][software-ml-llm-inference-tutorial] and [LLM Fine-tuning][software-ml-llm-fine-tuning-tutorial] first, as this will build upon them.
 
 ### Set up Podman
 
-If not already done as part of the [LLM Inference tutorial][ref-mlp-llm-inference-tutorial], edit your podman configuration in `$HOME/.config/containers/storage.conf` as follows:
+If not already done as part of the [LLM Inference tutorial][software-ml-llm-inference-tutorial], edit your podman configuration in `$HOME/.config/containers/storage.conf` as follows:
 
 ```toml title="$HOME/.config/containers/storage.conf"
 [storage]
```

Comment:
this path is different if you use XDG variables

Reply:
Will be included in next commit.